toolkit/resources/manifests/package/pkggen_core_x86_64.txt

```diff
@@ -161,6 +161,6 @@ libffi-3.2.1-12.cm1.x86_64.rpm
 libtasn1-4.14-2.cm1.x86_64.rpm
 p11-kit-0.23.16.1-2.cm1.x86_64.rpm
 p11-kit-trust-0.23.16.1-2.cm1.x86_64.rpm
-ca-certificates-shared-20200720-9.cm1.noarch.rpm
-ca-certificates-tools-20200720-9.cm1.noarch.rpm
-ca-certificates-base-20200720-9.cm1.noarch.rpm
+ca-certificates-shared-20200720-10.cm1.noarch.rpm
+ca-certificates-tools-20200720-10.cm1.noarch.rpm
+ca-certificates-base-20200720-10.cm1.noarch.rpm
```

Comment:
If you don't want `ca-certificates-static` in the worker chroot, you'll also need to update `toolkit/resources/manifests/package/update_manifests.sh`.